.travis.yml

```diff
@@ -35,13 +55,23 @@ jobs:
       # Docs and lint python versions must be synced with those in tox.ini
     - python: '3.8'
       env: TOX_ENV=docs
+    - python: '3.8'
+      arch: arm64
+      env: TOX_ENV=docs
     - python: '3.8'
       env: TOX_ENV=lint
   allow_failures:
     # Currently failing due to https://foss.heptapod.net/pypy/cffi/issues/458
     - python: nightly
 
 install:
+    - |
+      if [[ -n ${BUILD_WHEEL} ]]; then
+         pip install -U cibuildwheel
+         cibuildwheel --output-dir dist
+         rm -rf dist/*manylinux2010_x86_64.whl
```

Comment:
Why is this line necessary? Why is it only on linux and how would I know when/if the 2010 or other parts of the line need to change?

Reply:
I think this question is still relevant, and now "sudo" has also been added to just this command, raising the same question: why is it needed?